config.go

```diff
@@ -41,6 +43,18 @@ var (
 			regexp.QuoteMeta(tmplEnd),
 		),
 	)
+	QueryMetricsLabels = []string{"sql_job", "query"}
+	queryCounter       = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: fmt.Sprintf("%s_queries_total", metricsPrefix),
+	}, QueryMetricsLabels)
+	failedQueryCounter = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: fmt.Sprintf("%s_query_failures_total", metricsPrefix),
+	}, QueryMetricsLabels)
+
+	// Those are the default buckets
+	DefaultQueryDurationHistogramBuckets = []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 30, 60, 120}
```

Comment:
Do you think it would make sense here to use [prometheus.DefBuckets](https://github.com/prometheus/client_golang/blob/v1.19.0/prometheus/histogram.go#L265) for consistency?